samples/widgets/toggle.cpp

```diff
@@ -501,6 +500,7 @@ void ToggleWidgetsPage::CreateToggle()
 
 void ToggleWidgetsPage::AddButtonToSizer()
 {
+#ifdef wxHAS_BITMAPTOGGLEBUTTON
```

Comment:
Sorry for breaking this.
With this change `m_toggle` is never added to the sizer. I think guards should only be placed around if `m_chkFit->GetValue() )` and the `else` part.

A better fix would be to move  `m_chkFit` and `m_chkDisable` outside the guard, because these are still available when `wxHAS_BITMAPTOGGLEBUTTON` is not defined.

Reply:
No problem :-) Do you want me to cancel the pull request or would you rather I made the changes?